deployment/constructor_params/mainnet/root.yml

```
@@ -0,0 +1,52 @@
+deployment:
+  name: mainnet-root
+  chain_id: 1 # Ethereum Mainnet
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: mainnet.json
+
+constants:
+  # Threshold Network - References:
+  # - https://docs.threshold.network/resources/contract-addresses/mainnet/threshold-dao
+  T_TOKEN_ETH_MAINNET: "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5"
+  T_STAKING_CONTRACT: "0x01B67b1194C75264d06F808A921228a95C765dd7"
+  THRESHOLD_COUNCIL_ETH_MAINNET: "0x9F6e831c8F8939DC0C830C6e492e7cEf4f9C2F5f"
+
+  # FxPortal addresses - References:
+  # - https://github.com/0xPolygon/fx-portal#deployment-addresses
+  # - https://github.com/0xPolygon/fx-portal/blob/main/config/config.json
+  # - https://wiki.polygon.technology/docs/pos/design/bridge/l1-l2-communication/state-transfer/#prerequisites
+  FXPORTAL_CHECKPOINT_MANAGER: "0x86e4dc95c7fbdbf52e33d563bbdb00823894c287"
+  FXPORTAL_FXROOT: "0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2"
+  POLYGON_CHILD: "invalid" # Temporary invalid value that will be overwritten by workaround in deployment script
+
+  # TACo specific constants:
+  IN_SECONDS_1_DAY: 86400
+  IN_SECONDS_182_DAYS: 15724800
+  IN_SECONDS_364_DAYS: 31449600
+  IN_SECONDS_AVERAGE_MONTH_DURATION: 2628000  # 365*24*60*60/12
+  FORTY_THOUSAND_TOKENS_IN_WEI_UNITS: 40000000000000000000000
+  TIMESTAMP_FOR_2023_12_31_1100_UTC: 1703930400
```

As Derek mentioned, the timestamp `1703930400` corresponds to 2023 12 31 10:00 UTC. So we should change the variable's name
```suggestion
  TIMESTAMP_FOR_2023_12_31_1000_UTC: 1703930400
```

---

Are we 100% sure that's the timestamp we want?